### PR TITLE
Add Contributors process for papers

### PR DESCRIPTION
**Pull request by lumjjb**
During initial Cloud Native Security Whitepaper, it was discussed how the attribution and authorship should be carried out for papers produced by the STAG.

### CODE REVIEW

File: governance/paper-process.md [hunk 1]
```diff
@@ -72,18 +75,23 @@ Once task assignment is complete, contributors can begin content rough-in.
...
 Assignees can provide content rough-in in a variety of ways.  They may provide
-raw content in the form of phrases, disjointed paragraphs, bullets on the
-topic, or they draft content in the form of completed paragraphs.  If leveraging
-draft content, placing this content in quotes helps other reviewers know it is
-'near-complete' and can review it as a wholistic section.
-#### Opinions
+raw content in the form of phrases, disjointed paragraphs, bullets on the topic,
+or they draft content in the form of completed paragraphs.  If leveraging draft
+content, placing this content in quotes helps other reviewers know it is
+'near-complete' and can review it as a wholistic section.  #### Opinions
```
Comment (PushkarJ) [inline governance/paper-process.md:81]:
```suggestion
'near-complete' and can review it as a holistic section.  
#### Opinions
```

File: governance/paper-process.md [hunk 2]
```diff
@@ -72,18 +75,23 @@ Once task assignment is complete, contributors can begin content rough-in.
...
+the decision up to the reader.
 
 #### References
 
-It is important that any references used to generate paper content be cited appropriate through footnotes and a references section as appropriate.  This ensures the reader may not only dig deep into a specific reference area but also provides source content and is a matter of good practice.
-### Collaborative review
+It is important that any references used to generate paper content be cited
+appropriate through footnotes and a references section as appropriate.  This
+ensures the reader may not only dig deep into a specific reference area but also
+provides source content and is a matter of good practice.  ### Collaborative
```
Comment (PushkarJ) [inline governance/paper-process.md:93]:
```suggestion
provides source content and is a matter of good practice.  

### Collaborative
```

File: governance/paper-process.md [hunk 3]
```diff
@@ -173,3 +209,20 @@ strongly recommended the paper lead coordinate with TAG leadership to engage the
...
+given feedback during the Request for Comment (RFC) period of during the
+creation of the document are considered "Reviewers" of the project.
+
+Individuals/groups who have made huge contributions/impact on the work can be
+acknowledged in an "Acknowledgements" section. This is to highlight stellar
+contributions and commitments to individuals that have went above and beyond to
+contribute to the project.
+
+Each document should contain a "Contributors", "Reviewers" and
+"Acknowledgements" section where appropriate.  appropriate.
```
Comment (PushkarJ) [inline governance/paper-process.md:228]:
```suggestion
"Acknowledgements" section where appropriate. 
```

File: governance/paper-process.md [hunk 4]
```diff
@@ -173,3 +209,20 @@ strongly recommended the paper lead coordinate with TAG leadership to engage the
 CNCF team for posting a blog to summarize and link to the paper.  As community
 events occur, it is also recommended that the TAG coordinate a submission to
 community events (presentation) on the paper.
+
+#### Authorship, attribution and acknowledgements
```
Comment (PushkarJ) [inline governance/paper-process.md:213]: Thank you for addressing a topic that could get thorny in a succinct and fair way!

**Review by PushkarJ — COMMENTED** [review]
Few minor typos or newline edits - otherwise looks good!

**Comment by lumjjb** [comment]
Sad, the --fix flag failed me :(

File: governance/paper-process.md [hunk 5]
```diff
@@ -72,18 +75,27 @@ Once task assignment is complete, contributors can begin content rough-in.
...
 
-It is important that any references used to generate paper content be cited appropriate through footnotes and a references section as appropriate.  This ensures the reader may not only dig deep into a specific reference area but also provides source content and is a matter of good practice.
-### Collaborative review
+It is important that any references used to generate paper content be cited
+appropriate through footnotes and a references section as appropriate.  This
+ensures the reader may not only dig deep into a specific reference area but also
+provides source content and is a matter of good practice.  
+
+### Collaborative
+review
```
Comment (lumjjb) [inline governance/paper-process.md:98]:
```suggestion
```

File: governance/paper-process.md [hunk 6]
```diff
@@ -173,3 +212,20 @@ strongly recommended the paper lead coordinate with TAG leadership to engage the
...
+given feedback during the Request for Comment (RFC) period of during the
+creation of the document are considered "Reviewers" of the project.
+
+Individuals/groups who have made huge contributions/impact on the work can be
+acknowledged in an "Acknowledgements" section. This is to highlight stellar
+contributions and commitments to individuals that have went above and beyond to
+contribute to the project.
+
+Each document should contain a "Contributors", "Reviewers" and
+"Acknowledgements" section where appropriate.
```
Comment (lumjjb) [inline governance/paper-process.md:231]:
```suggestion
"Acknowledgements" section where appropriate.
```

File: governance/paper-process.md [hunk 7]
```diff
@@ -43,21 +43,24 @@ explained further in this document.
...
+
 * Who do we expect to read the paper?
 * What positions do they hold?
 * Why are we writing this paper?
 * What problem are we intending to resolve?
-* What assumptions are we making about the audience or the expected content of the paper?
-    * Note - these are documented then in the "Introduction > Assumptions" section of the paper
+* What assumptions are we making about the audience or the expected content of
+  the paper?
+  * Note - these are documented then in the "Introduction > Assumptions"
```
Comment (TheFoxAtWork) [inline governance/paper-process.md:59]:
```suggestion
  * Note - these are documented in the "Introduction > Assumptions"
```

File: governance/paper-process.md [hunk 8]
```diff
@@ -43,21 +43,24 @@ explained further in this document.
...
+      section of the paper
 * What are our Goals and Non-Goals?
-    * Note - these are documented then in the "Introduction > Scope > Goals" and "Introduction > Scope > Non-Goals"
+  * Note - these are documented then in the "Introduction > Scope > Goals" and
```
Comment (TheFoxAtWork) [inline governance/paper-process.md:62]:
```suggestion
  * Note - these are documented in the "Introduction > Scope > Goals" and
```

File: governance/paper-process.md [hunk 9]
```diff
@@ -72,18 +75,26 @@ Once task assignment is complete, contributors can begin content rough-in.
 ### Content rough-in
 
 Assignees can provide content rough-in in a variety of ways.  They may provide
-raw content in the form of phrases, disjointed paragraphs, bullets on the
-topic, or they draft content in the form of completed paragraphs.  If leveraging
-draft content, placing this content in quotes helps other reviewers know it is
-'near-complete' and can review it as a wholistic section.
+raw content in the form of phrases, disjointed paragraphs, bullets on the topic,
+or they draft content in the form of completed paragraphs.  If leveraging draft
```
Comment (TheFoxAtWork) [inline governance/paper-process.md:79]:
```suggestion
or draft content in the form of completed paragraphs.  If leveraging draft
```

File: governance/paper-process.md [hunk 10]
```diff
@@ -72,18 +75,26 @@ Once task assignment is complete, contributors can begin content rough-in.
...
+content, placing this content in quotes helps other reviewers know it is
+'near-complete' and can review it as a holistic section.
```
Comment (TheFoxAtWork) [inline governance/paper-process.md:81]:
```suggestion
'near-complete' and it should be reviewed it as a holistic section.  
```

File: governance/paper-process.md [hunk 11]
```diff
@@ -72,18 +75,26 @@ Once task assignment is complete, contributors can begin content rough-in.
...
+content, placing this content in quotes helps other reviewers know it is
+'near-complete' and can review it as a holistic section.  
+
 #### Opinions
 
-There may be situations where multiple differing views on the subject matter or controls implementation are expressed.  If these cannot be collaboratively resolved it is the duty of the authors to capture the different views and leave the decision up to the reader.
+There may be situations where multiple differing views on the subject matter or
+controls implementation are expressed.  If these cannot be collaboratively
+resolved it is the duty of the authors to capture the different views and leave
+the decision up to the reader.
```
Comment (TheFoxAtWork) [inline governance/paper-process.md:88]:
```suggestion
controls implementation are expressed.  Authors should make every effort to 
discover the industry standard or practice and document it. If none exists 
and the suggested content cannot be collaboratively resolved, it is the duty 
of the authors to capture the different views with references, benefits, and 
drawbacks or contingencies in order to provide the reader with enough 
information to leave the decision up to the reader.
```

File: governance/paper-process.md [hunk 12]
```diff
@@ -72,18 +75,26 @@ Once task assignment is complete, contributors can begin content rough-in.
...
 
 #### References
 
-It is important that any references used to generate paper content be cited appropriate through footnotes and a references section as appropriate.  This ensures the reader may not only dig deep into a specific reference area but also provides source content and is a matter of good practice.
-### Collaborative review
+It is important that any references used to generate paper content be cited
+appropriate through footnotes and a references section as appropriate.  This
```
Comment (TheFoxAtWork) [inline governance/paper-process.md:96]:
```suggestion
appropriately through footnotes and within the references section.  This
```

File: governance/paper-process.md [hunk 13]
```diff
@@ -72,18 +75,29 @@ Once task assignment is complete, contributors can begin content rough-in.
...
 #### Opinions
 
-There may be situations where multiple differing views on the subject matter or controls implementation are expressed.  If these cannot be collaboratively resolved it is the duty of the authors to capture the different views and leave the decision up to the reader.
+There may be situations where multiple differing views on the subject matter or
+controls implementation are expressed.  Authors should make every effort to 
+discover the industry standard or practice and document it. If none exists 
+and the suggested content cannot be collaboratively resolved, it is the duty 
+of the authors to capture the different views with references, benefits, and 
+drawbacks or contingencies in order to provide the reader with enough 
+information to leave the decision up to the reader.
```
Comment (lumjjb) [inline governance/paper-process.md:91]:
```suggestion
information to reach their own conclusion.
```

File: governance/paper-process.md [hunk 14]
```diff
@@ -72,18 +75,26 @@ Once task assignment is complete, contributors can begin content rough-in.
...
+resolved it is the duty of the authors to capture the different views and leave
+the decision up to the reader.
 
 #### References
 
-It is important that any references used to generate paper content be cited appropriate through footnotes and a references section as appropriate.  This ensures the reader may not only dig deep into a specific reference area but also provides source content and is a matter of good practice.
-### Collaborative review
+It is important that any references used to generate paper content be cited
+appropriate through footnotes and a references section as appropriate.  This
+ensures the reader may not only dig deep into a specific reference area but also
```
Comment (TheFoxAtWork) [inline governance/paper-process.md:97]:
```suggestion
permits the reader to not only dig deep into a specific reference area but also
```

File: governance/paper-process.md [hunk 15]
```diff
@@ -131,15 +145,15 @@ actively solicit public comment.
 Prior to opening the paper for community review, the group should select no more
 than 3 Adjudicators.  These individuals are tasked with accepting/rejecting
 suggestions, and resolving comments from the community.  Any comments or
-suggestions that require larger discussion should be brough up in a group
+suggestions that require larger discussion should be brought up in a group
 meeting and decisively resolved.
```
Comment (TheFoxAtWork) [inline governance/paper-process.md:152]:
```suggestion
meeting and decisively resolved. These should be clearly documented with justification in the notes.
```

File: governance/paper-process.md [hunk 16]
```diff
@@ -173,3 +212,20 @@ strongly recommended the paper lead coordinate with TAG leadership to engage the
...
+Papers and other resources created from TAG-Security efforts are under the
+authorship of TAG-Security, and all members who have made contributions to the
+document (through writing, editing, creating illustrations, etc.) are considered
+"Contributors". Members of the public or TAG-Security who have commented and
+given feedback during the Request for Comment (RFC) period of during the
+creation of the document are considered "Reviewers" of the project.
+
+Individuals/groups who have made huge contributions/impact on the work can be
+acknowledged in an "Acknowledgements" section. This is to highlight stellar
+contributions and commitments to individuals that have went above and beyond to
```
Comment (TheFoxAtWork) [inline governance/paper-process.md:230]:
```suggestion
contributions and commitments by individuals that have went above and beyond to
```

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
a few nits and 1 significant update that needs committed.

**Comment by lumjjb** [comment]
Made the changes, will do a full lint fix once all changes are final

**Review by TheFoxAtWork — APPROVED** [review]
LGTM

**Pull request merged**